go.mod

```diff
@@ -12,17 +12,18 @@ require (
 	github.com/aliyun/alibaba-cloud-sdk-go v1.60.340
 	github.com/aliyun/aliyun-oss-go-sdk v2.0.1+incompatible
 	github.com/aws/aws-sdk-go v1.21.10
+	github.com/baiyubin/aliyun-sts-go-sdk v0.0.0-20180326062324-cfa1a18b161f // indirect
 	github.com/coreos/go-systemd v0.0.0-20190719114852-fd7a80b32e1f
-	github.com/gardener/gardener v0.34.0
-	github.com/gardener/gardener-resource-manager v0.8.1
+	github.com/gardener/gardener v0.35.1-0.20200128130120-5b69a02f511a
+	github.com/gardener/gardener-resource-manager v0.9.1-0.20200124091350-6ea41bbae81f
 	github.com/gardener/machine-controller-manager v0.25.1-0.20200115123605-0510de7ddfca // master
 	github.com/go-logr/logr v0.1.0
 	github.com/gobuffalo/packr v1.25.0
 	github.com/gobuffalo/packr/v2 v2.1.0
 	github.com/golang/mock v1.3.1
 	github.com/gophercloud/gophercloud v0.7.0
 	github.com/gophercloud/utils v0.0.0-20190527093828-25f1b77b8c03
-	github.com/huandu/xstrings v1.2.0
+	github.com/huandu/xstrings v1.3.0
 	github.com/json-iterator/go v1.1.9 // indirect
 	github.com/onsi/ginkgo v1.8.0
```

Comment:
Nit: we can remove the `replace` for `github.com/census-instrumentation/opencensus-proto` as we had to introduce it because of the dependency cycle between g/g and g/gext.